### ISSUE TITLE
Filtering with multiple values

### ISSUE THREAD

**Issue by RoYoMi** [CrowdStrike/psfalcon#275]
Using powershell module PSFalcon v2.2.4, powershell v7

I'm trying follow along with this documentation: https://github.com/CrowdStrike/psfalcon/wiki/Filtering-Results

The doc says the filter must be a string, and multiple values can be included as a quote-comma delimited list inside square brackets.

In this image we can see my individual hosts will return an ID, but when I try doing this as prescribed nothing gets returned.

![image](https://user-images.githubusercontent.com/8366040/215194100-854c257b-7e86-4aaa-9a5e-c69849c5ecc8.png)

Is there something I'm doing wrong here?

**Comment by bk-cs** [comment]
Yes, multiple filters are joined with commas (OR) or plus (AND):

```
# hostname1 or hostname2
Get-FalconHost -Filter "hostname:'hostname1',hostname:'hostname2'"
```
```
# hostname1 that is Windows
Get-FalconHost -Filter "hostname:'hostname1'+platform_name:'Windows'"
```
